handbook/ce/customer-support-onboarding.md

```diff
@@ -1,5 +1,204 @@
+
+Big pause here. These next tasks need a ton of improvement -- as mentioned in the last bullet! They are all supposed to help you learn the product and get more comfortable doing the job. Work with what is here and take abundant notes for how this can be improved.
+
+* Browse some of the available materials to help you learn more about the product:
+	* [Campaigns Overview Deck](https://docs.google.com/presentation/d/1CN3KQf1Hfdb4RO6FgBgKuiHK4ERcOAHPgVnOcBu-MPU/edit#slide=id.g7d2aea8729_0_0)
+	* [Structural Search Overview](https://zoom.us/rec/share/CJtwQ7uEp3v1pvPqdUD7GDuaYm_2g6w3zSP7GNA3aGQHZDjQ72awYXvHEnwsoio6.Bt-0DuuAZjs7UXMc?startTime=1606237440000)
+	* Code Intelligence [video](https://drive.google.com/file/d/1TyCj62LLmhvamXK-CC6D8-7uGk1jmsKj/view) and [slides](https://docs.google.com/presentation/d/181oMTXRmcTqTCfOe5P__fLKwlzV1uk9wvwG6ocFgz20/edit?usp=sharing)
+	* [“Deployment types - A crash course in Sourcegraph”](https://docs.google.com/presentation/d/1u4mbXjubQqV-6WFbuS7Q1b_X6BVh-_GWzzFQMcrAzLw/edit#slide=id.p)
+	* [Architecture overview](https://docs.sourcegraph.com/dev/background-information/architecture)
+	* [https://kube.academy/](https://kube.academy/)
+* Read about [search queries](https://docs.sourcegraph.com/code-search) and perform your first searches.
+* Work through the questions from the [Sales Onboarding Quiz](https://about.sourcegraph.com/handbook/sales/onboarding/quiz) to make sure you understand key concepts. Feel free to skip any obvious answers and discuss any questions you have in our #customer-support-chat Slack channel.
+* Explore our official [documentation](https://docs.sourcegraph.com/getting-started). Get used to navigating it and trying to find what you need in it. We know we have a lot of opportunity for improvement -- more on that soon!
+* Explore the sections of the handbook for each engineering team. Get used to trying to find answers in the handbook as much as our documentation. 
+	* [Distribution](https://about.sourcegraph.com/handbook/engineering/distribution) 
+	* [Web](https://about.sourcegraph.com/handbook/engineering/web)
+	* [Campaigns](https://about.sourcegraph.com/handbook/engineering/campaigns)
+	* [Core application](https://about.sourcegraph.com/handbook/engineering/core-application)
+	* [Code insights](https://about.sourcegraph.com/handbook/engineering/code-insights)
+	* [Code intelligence](https://about.sourcegraph.com/handbook/engineering/code-intelligence)
+	* [Search](https://about.sourcegraph.com/handbook/engineering/search)
+	* [Security](https://about.sourcegraph.com/handbook/engineering/security)
```

Comment:
Could also be relative links—I'm going to stop leaving this comment and I don't super know that it matters, it's just a style thing I've seen in the rest of the handbook.

Reply:
I'll go through and clean these up soon!